atecc608a/main.c

```diff
@@ -96,11 +96,12 @@ psa_key_slot_number_t atecc608a_public_key_slot = 9;
 enum {
     key_type = PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_CURVE_SECP256R1),
     keypair_type = PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_CURVE_SECP256R1),
-    key_bits = 256,
+    priv_bits = PSA_BYTES_TO_BITS(32),
+    pub_bits = PSA_BYTES_TO_BITS(64),
     hash_alg = PSA_ALG_SHA_256,
     alg = PSA_ALG_ECDSA(hash_alg),
-    sig_size = PSA_ASYMMETRIC_SIGN_OUTPUT_SIZE(key_type, key_bits, alg),
-    pubkey_size = PSA_KEY_EXPORT_ECC_PUBLIC_KEY_MAX_SIZE(key_bits),
+    sig_size = PSA_ASYMMETRIC_SIGN_OUTPUT_SIZE(key_type, priv_bits, alg),
+    pubkey_size = PSA_KEY_EXPORT_ECC_PUBLIC_KEY_MAX_SIZE(pub_bits),
```

Comment:
Looking at the `PSA_KEY_EXPORT_ECC_PUBLIC_KEY_MAX_SIZE` macro and the `docs/getting_started.md` from mbed crypto - it looks like it should compute the size from priv_bits.